language/stdlib/modules/LibraConfig.move

```diff
@@ -73,6 +73,33 @@ module LibraConfig {
         reconfigure_();
     }
 
+    // Set a config item to a new value and trigger a reconfiguration.
+    public fun set_with_capability<Config: copyable>(
+        _cap: &ModifyConfigCapability<Config>,
+        payload: Config
+    ) acquires LibraConfig, Configuration {
+        let addr = CoreAddresses::LIBRA_ROOT_ADDRESS();
+        assert(exists<LibraConfig<Config>>(addr), ECONFIG_DOES_NOT_EXIST);
+        let config = borrow_global_mut<LibraConfig<Config>>(addr);
+        config.payload = payload;
+        reconfigure_();
+    }
+
+    // Publish a new config item. The caller will use the returned ModifyConfigCapability to specify the access control
+    // policy for who can modify the config.
+    public fun publish_new_config_with_capability<Config: copyable>(
```

Comment:
the name is confusing, it's not publish with capability, it's publish and return the capability, this probably can be consolidated with `publish_new_config`

Reply:
Great point!!!
I'll rename this to `publish_new_config_and_get_capability` and make the `publish_new_config` call it to avoid code duplication.